java/src/main/java/io/cucumber/gherkin/StringUtils.java

```diff
@@ -114,4 +115,21 @@ private static boolean contains(char[] characters, char c) {
         return false;
     }
 
+    public static class TrimmedText {
+        private final String text;
+        private final int indent;
+
+        public TrimmedText(String text, int indent) {
```

Comment:
Complete nitpick. But can you flip these arguments? That way from left to right reads  "indent - text".